tests/httpServer/server.test.ts

```diff
@@ -259,172 +259,172 @@ describe("httpserver-tests", () => {
       request: { url: "/handler/bob" },
     });
   });
+});
 
-  async function testAuthMiddlware(ctx: MiddlewareContext) {
-    if (ctx.requiredRole.length > 0) {
-      const { userid } = ctx.koaContext.request.query;
-      const uid = userid?.toString();
-
-      if (!uid || uid.length === 0) {
-        const err = new DBOSNotAuthorizedError("Not logged in.", 401);
-        throw err;
-      } else {
-        if (uid === "go_away") {
-          throw new DBOSNotAuthorizedError("Go away.", 401);
-        }
-        return Promise.resolve({
-          authenticatedUser: uid,
-          authenticatedRoles: uid === "a_real_user" ? ["user"] : ["other"],
-        });
+async function testAuthMiddlware(ctx: MiddlewareContext) {
```

Comment:
What are the changes here? They don't seem directly related to the auto-registration

Reply:
This is 98% a whitespace change, provoked by a linter irritation.  It thought the classes below became unused because they weren't mentioned anywhere (had been in `[]`).  One trick I used was to put `export` on the classes, but in this case that didn't work because they were inside the function, so I moved them to top-level.

Reply:
Yeah so what happened with the linter? What's the "right" way to write a entrypoint file now?

Reply:
The linter thought the classes were unused (they are only used by the decorator registering them, never called out by name).  You can either mark them with a linter comment, or export them.  I think exporting them is typical of a user app, so I did that, but if preferred I can silence the linter for those classes.

Reply:
Oh... the entrypoint file.  That needn't change... this is all the test runtime, which works differently.

Reply:
Anyway, reverted the 2 with the whitespace changes, so now the PR is smaller and those test the backward compatibility.